openmrs/apps/registration/app.json

```diff
@@ -116,16 +116,15 @@
             "fieldValidation" : {
                 "primaryContact" : {"pattern" : "[0-9]{8,10}", "errorMessage" : "Should be 12 characters starting with 91 (country code)"},
                 "landHolding" : {"pattern" : "[0-9]{0,3}", "errorMessage" : "Should be between 1 to 999 acres"},
-                "familyName" : {"pattern" : "[a-zA-Z]{0,}", "errorMessage" : "Should contain characters"},
-                "givenName" : {"pattern" : "[a-zA-Z]{0,}", "errorMessage" : "Should contain characters"},
-                "middleName" : {"pattern" : "[a-zA-Z]{0,}", "errorMessage" : "Should contain characters"},
-                "latitude" : {"pattern" : "[a-zA-Z]{0,}", "errorMessage" : "Should contain characters"},           
+                "familyName" : {"pattern" : "[a-záàãâéèêẽíìóòõôúùçA-ZÁÀÃÂÉÈÊẼÍÌÓÒÔÕÚÙÇ]{0,}", "errorMessage" : "Should contain characters"},
+                "givenName" : {"pattern" : "[a-záàãâéèêẽíìóòõôúùçA-ZÁÀÃÂÉÈÊẼÍÌÓÒÔÕÚÙÇ]{0,}", "errorMessage" : "nome Should contain characters"},
```

Comment:
@simaomazive should we be adding `nome` to the label here?

Reply:
Ohw No @nthfloor was i mistake. I'll remove...